llvm/test/Transforms/DeadStoreElimination/stores-of-existing-values.ll

```diff
@@ -655,3 +655,81 @@ exit:
   call void @use(ptr %p) argmemonly
   ret void
 }
+
+define void @scalable_scalable_redundant_store(ptr %ptr) {
+; CHECK-LABEL: @scalable_scalable_redundant_store(
+; CHECK-NEXT:    [[GEP:%.*]] = getelementptr i64, ptr [[PTR:%.*]], i64 2
+; CHECK-NEXT:    store <vscale x 2 x i64> zeroinitializer, ptr [[GEP]], align 16
+; CHECK-NEXT:    store <vscale x 4 x i64> zeroinitializer, ptr [[PTR]], align 32
+; CHECK-NEXT:    ret void
+;
+  %gep = getelementptr i64, ptr %ptr, i64 2
```

Comment:
nit: adjusting the name to include the offset may make things slightly easier to read.

```suggestion
  %gep.ptr.2 = getelementptr i64, ptr %ptr, i64 2
```